Libraries/pbxspec/Sources/Manager.cpp

```diff
@@ -86,9 +86,13 @@ findSpecification(std::vector<std::string> const &domains, std::string const &id
 {
     typename T::vector vector = findSpecifications <T> (domains, type);
```

Comment:
Another option would be to pull out `pbxbuild::DirectedGraph<T>` and use the toposort from that rather than the current brute force DFS. That has error handling and unit tests, unlike the current specification inheritance code. 😭